src/mame/machine/midwayic.cpp

```diff
@@ -9,13 +9,25 @@
 #include "emu.h"
 #include "midwayic.h"
 
-
-#define LOG_NVRAM           (0)
-
+#define LOG_IRQ             (1U << 0)
+#define LOG_NVRAM           (1U << 1)
+#define LOG_FIFO            (1U << 2)
+#define LOG_IOASIC          (1U << 3)
+#define LOG_PIC             (1U << 4)
+#define LOG_UART            (1U << 5)
```

Comment:
You can’t use `1U << 0` because it conflicts with the default definition for `LOG_GENERAL` used for the `LOG` macro.